benchmarks/aes/go/server.go

```diff
@@ -0,0 +1,160 @@
+// MIT License
+
+// Copyright (c) 2022 EASE lab
+
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+// Package main implements a server for Greeter service.
+package main
+
+import (
+	"context"
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"encoding/hex"
+	"flag"
+	"fmt"
+	"io"
+
```

Comment:
I don't think that this file was linted, please investigate why the go linter check didn't complain.

Reply:
I'm not sure which linter you are referring to. Is it [LinkCheck](https://github.com/ease-lab/vSwarm/runs/5691233038?check_suite_focus=true)? That only runs on Markdown, not on any go files.

Reply:
ok, this repo does not have a linter. please add for this workload as a step in the workflow. [guide](https://freshman.tech/linting-golang/). highly recommended when writing any code

Reply:
@harshitgarg22 do you use vs code for coding?
There is a go extention. Install it and all the go tools that come with it.
The every time you save the file gets automatically linted